README.rst

```diff
@@ -21,10 +21,35 @@ Clone the repository, and pip install::
    pip install -e .
 
 Copy the example config file to be visible for starting up a test database::
-   
+
    cp ./knimin/config.txt.example ./knimin/config.txt
 
-You should now be able to start the webserver::
+**Installing JIRA**
+
+Note that these instructions should work for Mac or Linux and that
```

Comment:
But we do ask the user to install conda and create an environment in the beginning of the readme. Therefore, I'd say lets use conda as the default and pro users like you can derive whatever is necessary to install in the root environment, i.e. without conda.
It would also be easier to remove the whole environment later on your machine.